src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1589,6 +1590,27 @@ void requestsShouldChangePositionWhenTheyGoInFulfillmentOnCheckIn() {
       allOf(isOpenAwaitingPickup(), hasPosition(2)));
   }
 
+  @Test
+  void shouldNotThrowNPEWhenCheckInNotExistingItemInRequestQueue() {
```

Comment:
Which item is "NotExisting"? From the test I can see that both items exist. Or did you mean that there is no request for the second item in the queue?
Also, mentioning an exception in the test name does not seem to provide much information about the expected result of the test. Can we think of a better name?

Reply:
agree, done